docs/project-charter.md

```diff
@@ -0,0 +1,196 @@
+of expertise, balanced with the ability to make decisions efficiently. The TSC
+must have at least four members. Although there are no hard requirements, the
+composition of the TSC should aim for geographic and employer diversity in the
+spirit of a distributed maintenance model.
+
+## Section 4: Roles & Responsibilities
+
+_directions: describe the roles and responsibilities of the LoopBack Governing Body._
+
+ex. [K8s SIG Architecture Charter](https://github.com/kubernetes/community/blob/HEAD/sig-architecture/charter.md#roles-and-organization-management)
+ex. [Node.js TSC Charter](https://github.com/nodejs/TSC/blob/HEAD/TSC-Charter.md#section-4-responsibilities-of-the-tsc)
+
+The LoopBack TSC is responsible for:
+
+- Release dates
+- Creating new releases
+- Release quality standards
+- Technical direction
+- GitHub repository hosting
+- Infrastructure management
+- License and security compliance
+- Conduct guidelines and enforcement
+- Maintaing the list of LoopBack Maintainers
+- Mediating technical conflicts between Collaborators or Foundation projects
+- Hosting and publishing the monthy LoopBack Maintainers Call
+
+### Section 4.1 Project Operations & Management (optional)
+
+_directions: use this section to describe any other specific tasks the_
+_${PROJECT} Governing Body may be responsible for regarding process or project_
+_operations and management. If you are not using this section, please indicate_
+_your intent with the phrase, 'Section Intentionally Left Blank'._
+
+ex. [K8s SIG Architecture Charter](https://github.com/kubernetes/community/blob/HEAD/sig-architecture/charter.md#roles-and-organization-management)
+ex. [Node.js TSC Charter](https://github.com/nodejs/TSC/blob/HEAD/TSC-Charter.md#section-5-nodejs-project-operations)
+
+The TSC and entire technical community will follow any processes as may be
+specified by the OpenJS Foundation Board relating to the intake and license
+compliance review of contributions, including the OpenJS Foundation IP Policy.
+
+### Section 4.2: Decision-making, Voting, and/or Elections (optional)
+
+_directions: describe any provisions the project makes for decision-making_
+_or include the information by reference your governance.md document._
+_If you are not using this section, please indicate your intent with the_
+_phrase, 'Section Intentionally Left Blank'._
+
+ex. [Node.js TSC Charter](https://github.com/nodejs/TSC/blob/HEAD/TSC-Charter.md#section-6-elections)
+
+Unless stated otherwise, the LoopBack TSC adopts a lazy consensus voting system,
+where consensus is assumed when there are no outstanding objections from the
+other TSC members after a stipulated period of time of no less than 72 hours.
+
+For more important matters such as the nomination of new TSC members, a lazy
+consensus for a stipulated period of no less than 1 week is used instead.
+
+These votes must be done through a process that's accessible to the general
```

Comment:
For voting to approve/reject a contributor to maintainer, the votes are not public in the current process.

Reply:
> Should the TSC voting process be private as well?

sounds good as it falls under the category, IMHO.